tlatools/org.lamport.tlatools/src/tla2sany/semantic/LevelNode.java

```diff
@@ -96,13 +96,22 @@ public class LevelNode extends SemanticNode {
   * recursively defined operators or functions.                            *
   *************************************************************************/
 
+  /**
+   * Run the level-checking algorithm on this node and all its children.
+   * @param errors A log in which to record any errors which occurred.
+   * @return Whether the level-checking process succeeded.
+   */
+  public boolean levelCheck(Errors levelCheckingErrorLog) {
```

Comment:
I used `errors` for all the internal methods so they would switch from using the static `SemanticNode.errors` field to using their local method scope's `errors` parameter; since this is a more public-facing method do you think it makes sense to give it a more descriptive name?

Reply:
Although you're right I did get the `@param` name wrong here